_includes/banner.html

```diff
@@ -3,7 +3,7 @@
     <header class="usa-banner__header">
       <div class="usa-banner__inner">
         <div class="grid-col-auto">
-          <img class="usa-banner__header-flag" src="{{ site.baseurl }}/assets/img/us_flag_small.png" alt="U.S. flag">
+          <img class="usa-banner__header-flag" src="{{ site.baseurl }}/assets/img/us_flag_small.png" alt="U.S. flag" width="16" height="11">
```

Comment:
Does this make a big difference for layout shift? If so, we should create a feature request in USWDS to add this to the component.

Reply:
It doesn't appear to have made a big difference in layout shift in this instance, but it does seem to be best practice (again). 

I have opened [this USWDS issue](https://github.com/uswds/uswds/issues/4831) so that we can return to the topic of adding height/width to our USWDS component image markup.

Reply:
Thanks for the clarification. I'm okay with including this since there's an issue to track it on USWDS.